tests-perf/locust/locust-notifications.py

```diff
@@ -1,24 +1,88 @@
+""" locust-notifications.py
+   isort:skip_file
+"""
+# flake8: noqa
+
 import os
+import sys
+from datetime import datetime
+
+sys.path.append(os.path.abspath(os.path.join("..", "tests_smoke")))
 
 from dotenv import load_dotenv
 from locust import HttpUser, constant_pacing, task
+from tests_smoke.smoke.common import job_line, rows_to_csv  # type: ignore
 
 load_dotenv()
-AUTH_HEADER = os.getenv("TEST_AUTH_HEADER")
 
 
 class NotifyApiUser(HttpUser):
 
     wait_time = constant_pacing(60)
-    host = "https://api.staging.notification.cdssandbox.xyz"
+    host = os.getenv("LOAD_TEST_DOMAIN", "https://api.staging.notification.cdssandbox.xyz")
+
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": os.getenv("TEST_AUTH_HEADER")}
+        self.email = "success@simulator.amazonses.com"
+        self.template_id = "5ebee3b7-63c0-4052-a8cb-387b818df627"
+
+    @task(1)
```

Comment:
I would give more weight to this task given it's the one that is the most used.

Reply:
This[1] suggests to my understanding that `@task(1)` has the highest chance of getting called

[1] https://docs.locust.io/en/stable/writing-a-locustfile.html#task-decorator

Reply:
I believe it's the opposite - the higher the number, the more it's called. For example, when I ran the code with 100 users it was mostly the sms task (weight 16)
```
Name                                  # reqs     
--------------------------------------------
 POST /v2/notifications/bulk             23
 POST /v2/notifications/email            19
 POST /v2/notifications/sms              58
--------------------------------------------
 Aggregated                             100
```
I'd say weight the `/email` at something like 10 and the rest at 1?